src/ripple/ledger/impl/PaymentSandbox.cpp

```diff
@@ -186,15 +186,26 @@ PaymentSandbox::balanceHook (AccountID const& account,
     {
         auto delta = amount.zeroed ();
         auto lastBal = amount;
+        auto minBal = amount;
         for (auto curSB = this; curSB; curSB = curSB->ps_)
         {
             if (auto adj = curSB->tab_.adjustments (account, issuer, currency))
             {
                 delta += adj->debits;
                 lastBal = adj->origBalance;
```

Comment:
Not a correction, just a question for my own understanding.

I'm assuming that several of the recursively referenced `Sandboxes` can each return a non-`boost::none` `adjustments (account, issuer, currency)`.  What kind of confidence do we have that all of them will return the same `adj->origBalance `?  If they can return different `adj->origBalance` values, how will that play in this code?